src/pyodide/internal/sphinx/conf.py

```diff
@@ -0,0 +1,38 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'Python Workers API'
+copyright = '2025, Cloudflare'
+author = 'Cloudflare'
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx_markdown_builder'
```

Comment:
We might want to add `sphinx.ext.autosummary` (can make a table with links to the individual documentation entries) and `myst_parser` so we can write the source files in markdown rather than rst. Also, `sphinx.ext.intersphinx` would be helpful, particularly if we can add xref links to cloudflare docs and mdn.
https://github.com/pyodide/pyodide/blob/main/docs/sphinx_pyodide/sphinx_pyodide/mdn_xrefs.py

None of these things have to happen in this PR though.